src/steamnetworkingsockets/clientlib/steamnetworkingsockets_connections.cpp

```diff
@@ -2660,7 +2660,7 @@ void CSteamNetworkConnectionBase::PostConnectionStateChangedCallback( ESteamNetw
 	// Send API callback for this state change?
 	// Do not post if connection state has not changed from an API perspective.
 	// And don't post notifications for internal connections used by ad-hoc-style endpoints.
-	if ( eOldAPIState != eNewAPIState && !m_pMessagesEndPointSessionOwner )
+	if ( eOldAPIState != eNewAPIState )
```

Comment:
Sorry, I am just now getting around to pulling this.....

The other change looks good.  However, I'm not sure about this one.  Do you remember why this change was necessary?  In general, connections that are owned by ISteamNetworkingMessages do not send feedback to the app through this callback.  Instead, SteamNetworkingMessagesSessionRequest_t and SteamNetworkingMessagesSessionFailed_t are used instead.

Reply:
Ooops, I had not handled those callbacks because they were not described in documentation.
Just checked - after handling SteamNetworkingMessagesSession* callbacks this commit is not required.